org.civicrm.reqangular/src/common/services/file-upload.js

```diff
@@ -1,48 +1,65 @@
 define([
   'common/modules/services',
+  'common/angularFileUpload'
 ], function (module) {
   'use strict';
 
-  module.factory('FileUploadService', ['$q', 'FileUploader',
+  module.factory('FileUpload', ['$q', 'FileUploader',
     function ($q, FileUploader) {
-      var fileUploader;
+
+      /**
+      * Helper to call throw on expressions by wrapping it on function
+      *
+      * @param {String} error message to throw
+      */
+      function _throw (msg) {
```

Comment:
throw msg cannot be called directly in expression like `entity_table: customSettings.entityTable || throw 'entityTable missing from custom settings'` will not work